blog/fine-grained-news-2025-02.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Fine Grained News - February 2025
+description: Fine Grained News
+slug: fine-grained-news-2025-02
+date: 2025-02-26
+authors: aaguiar
+tags: [newsletter]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Fine Grained News - February 2025
+
+Welcome to the second Fine Grained News edition of 2025! 
+
+## Just Shipped!
+
+- We shipped 3 minor versions of OpenFGA which include:
+  - A fix for [CVE-2025-25196](https://github.com/openfga/openfga/security/advisories/GHSA-g4v5-6f5p-m38j) and [CVE-2024-56323](https://github.com/openfga/openfga/security/advisories/GHSA-32q6-rr98-cjqv)
+  - Several performance improvements that are enabled with the `enable-check-optimizations` experimental flag.
+  - Dynamic TLS certificate reloading for HTTP and gRPC servers. Thanks [Rokibul Hasan](https://github.com/RokibulHasan7) for your contribution!
+  - A `name` filter to ListStores. The name parameter instructs the API to only include results that match that name. Thanks [Karl Persson](https://github.com/kalleep) for your contribution!
+  - Optimized database dialect handling by setting it during initialization instead of per-call, fixing SQL syntax errors in MySQL. Thanks [Siddhant Khare](https://github.com/Siddhant-K-code) for your contribution!
+
```

Comment:
There seems to be something off in the new lines and the indentation of the bullet points

Reply:
They look good on the web page